### PR TITLE
Add ShowMapper::removeGroup method

### PR DESCRIPTION
**Pull request by marcoslibre**
I am targetting this branch, because it is a new feature

## Changelog

```markdown
### Added
- Added `ShowMapper::removeGroup` method

```

## To do

- [x] Update the tests
- [x] Update the documentation

## Subject

Add ShowMapper::removeGroup method, similar to Add FormlMapper::removeGroup method

### CODE REVIEW

**Comment by greg0ire** [comment]
In which case is this useful?

**Comment by marcoslibre** [comment]
For example, it occurs when you extend existing Admins, and you might want to delete groups from the admin, or make them not show.

It is the same as the existing remove() method, but instead of removing just a field, you may want to remove a whole group (and/or a tab).

**Comment by greg0ire** [comment]
There should only be one commit in your PR. Please rebase interactively to get rid of the merge commit, and when you push, force push : when you rework a commit, you want to completely replace what is on github with your new commit, instead of merging both commits.

File: Show/ShowMapper.php [hunk 1]
```diff
@@ -114,6 +114,47 @@ public function remove($key)
     }
 
     /**
+     * Removes a group.
+     *
+     * @param string $group          The group to delete
+     * @param string $tab            The tab the group belongs to, defaults to 'default'
+     * @param bool   $deleteEmptyTab Whether or not the Tab should be deleted, when the deleted group leaves the tab empty after deletion
```
Comment (greg0ire) [inline Show/ShowMapper.php]: - the Tab => the parent tab
- "should be deleted" => "should be deleted too"
- it is ok to use linebreaks in comments, like this : 

```
* @param bool   $deleteEmptyTab Whether or not the parent tab should be deleted too, 
*                                                       when the deleted group leaves the tab empty after deletion
```

**Comment by marcoslibre** [comment]
Sorry for the rebase... I'm not very familiar with pull requests.
So, I made the changes you suggested in the PHPDoc, rebased and force pushed.
Is it correct ?

**Comment by greg0ire** [comment]
> Is it correct ?

There is one commit so the rebase is correct, congrats! Let's see the code.

**Comment by marcoslibre** [comment]
Where should I put the documentation ?
I guess it's in Ressources/doc, but where ?

**Comment by greg0ire** [comment]
What about in `Resources/doc/reference/action_show.rst`? Seems appropriate…

**Comment by marcoslibre** [comment]
OK, tests and doc added

**Comment by greg0ire** [comment]
Great job! LGTM

**Comment by greg0ire** [comment]
Can you check the unit tests please?

**Comment by marcoslibre** [comment]
I cleaned the unit tests

**Comment by OskarStark** [comment]
Good job @marcoslibre and thank you for the contribution 👍 🍺

**Pull request merged**